substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -1786,6 +1788,14 @@ impl<T: Config> Pallet<T> {
         }
     }
 
+    fn get_stash_balance(twin_id: u32) -> BalanceOf<T> {
+        let account_id = pallet_tfgrid::TwinBoundedAccountID::<T>::get(twin_id);
+        match account_id {
+            Some(account) => Self::get_usable_balance(&account),
```

Comment:
not sure if `get_usable_balance()` is the right one that should be used?

Reply:
I think that is fine yes. A stash account isn't used in billing and funds will probably never be locked there.